Misc/NEWS.d/next/Library/2023-01-02-16-59-49.gh-issue-100690.2EgWPS.rst

```diff
@@ -0,0 +1,3 @@
+Mock objects which are not unsafe will now raise an AttributeError when accessing an
+attribute that matches the name of an assertion but without the prefix ``assert_``, e.g. accessing ``called_once`` instead of ``assert_called_once``.
+This is in addition to this already happening for accessing attributes with prefixes assert, assret, asert, aseert, and assrt.
```

Comment:
Minor thing: can you wrap to 72 columns?  And add markup to Mock and AttributeError, plus code markup for the prefixes on last line.